cfme/tests/configure/test_db_backup_schedule.py

```diff
@@ -146,7 +170,7 @@ def test_db_backup_schedule(request, db_backup_data):
     # the dash is there to make strftime not use a leading zero
     hour = dt.strftime('%-H')
     minute = dt.strftime('%-M')
-
+    hostname = db_depot_machine_ip + db_backup_data.sub_folder
```

Comment:
We should no longer call this `hostname`. It should be `uri` or `url` instead.
Subsequently, the proper way to join URL components would be:
```
import urlparse
urlparse.urljoin("192.168.0.1", "/my_subfolder")
```